python/lsst/pipe/analysis/colorAnalysis.py

```diff
@@ -379,7 +410,7 @@ class ColorAnalysisTask(CmdLineTask):
     @classmethod
     def _makeArgumentParser(cls):
         parser = ArgumentParser(name=cls._DefaultName)
-        parser.add_id_argument("--id", "deepCoadd_forced_src",
+        parser.add_id_argument("--id", "deepCoadd_obj",
```

Comment:
shouldn't this be conditional on whether `self.config.doReadParquetTables`?  i.e., if not, then it should still look for `deepCoadd_forced_src`?